package.json

```diff
@@ -50,13 +52,31 @@
     "docs:watch": "gitbook install && gitbook serve",
     "prepublish": "nwb build",
     "release": "semantic-release pre && npm publish && semantic-release post",
-    "test": "node umd/skatejs.js && semistandard && npm run test:ts && nwb test -- --ci",
-    "test:ts": "tsfmt -r && tsc -p ./",
-    "test:watch": "nwb test --server"
+    "pretest": "npm run style && npm run test:node && npm run test:ts",
```

Comment:
speaking about small library and amount of npm scripts ... ehm https://github.com/developit/preact/blob/master/package.json#L10 :D 🍻